src/main/java/teammates/common/util/LogMessageGenerator.java

```diff
@@ -28,11 +29,11 @@
      * @param currUser login information generated by {@link teammates.logic.api.GateKeeper}
      * @param userAccount authentication user account generated by action
      * @param unregisteredStudent authentication unregisteredStudent attributes generated by action
-     * @param logMessage log message to show to admin
+     * @param logMessages log message to show to admin
```

Comment:
The description hasn't been updated yet

Reply:
`list of log message to show to admin` would this suffice ?

Reply:
Yes that's good